### PR TITLE
add distribution onboarding

### PR DESCRIPTION
**Pull request by davejrt**

### CODE REVIEW

File: handbook/engineering/distribution/onboarding.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+- Become familiar with Distribution [product](product.md),[tech stack](tech_stack.md) and [processes](recurring_processes.md).
+- Go through [General Onboarding](../../people-ops/onboarding/general_onboarding.md)
+- You have a sense of what tools you will be using and for what
+
+
+#### Tasks:
+- Meet your onboarding buddy
+- Add shared [Distribution calendar](https://calendar.google.com/calendar/u/0?cid=Y19qZHU3NTJ2anFpZ2NnMWVmYmYxZXIzY291Z0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) to your calendar
+- Attend weekly sync [meeting](https://calendar.google.com/event?action=TEMPLATE&tmeid=OXFyZXRsc21hdDh1MHNkNzQ2aGhqMzg3NmxfMjAyMTA1MjRUMTczMDAwWiBkYXZlQHNvdXJjZWdyYXBoLmNvbQ&tmsrc=dave%40sourcegraph.com&scp=ALL)
+- Deploy your own SG instance using one of the 3 deployment methods
```
Comment (daxmc99) [inline handbook/engineering/distribution/onboarding.md:21]: I would clarify this to created a personal GCP project to deploy into. 
Also, thoughts on doing docker-compose first then k8s?

File: handbook/engineering/distribution/onboarding.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  - #dev-accounce
+  - #alerts-distribution
+
+### Week 2
+
+#### Goals:
+- Develop high-level understanding of product.
+- You are participating in our team rituals.
+- Get to know your team.
+- Close at least one issue.
```
Comment (daxmc99) [inline handbook/engineering/distribution/onboarding.md:39]:
```suggestion
- Work towards closing one issue.
```
I don't think we want to make someone feel like they are falling behind if they don't close an issue by here (took me about 3 weeks to onboard and finish up the version warnings issues)

**Review by daxmc99 — APPROVED** [review]

File: handbook/engineering/distribution/onboarding.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
+# Distribution Engineer Oboarding
+
+Welcome to the distribution team! This document will guide you through distribution-specific onboarding tasks to make your start here at Sourcegraph as smooth as it can be.
+
+## Distribution Onboarding Checklist
+
+### Week 1
+
+#### Goals:
+- Clear knowledge of what the company’s, and Distributions’s mission, vision, values and culture are.
```
Comment (dan-mckean) [inline handbook/engineering/distribution/onboarding.md:10]: Add some links :)
https://about.sourcegraph.com/company/goals
https://about.sourcegraph.com/handbook/engineering/distribution
https://about.sourcegraph.com/company/values
https://about.sourcegraph.com/handbook/communication/code_of_conduct

File: handbook/engineering/distribution/onboarding.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
+# Distribution Engineer Oboarding
+
+Welcome to the distribution team! This document will guide you through distribution-specific onboarding tasks to make your start here at Sourcegraph as smooth as it can be.
```
Comment (dan-mckean) [inline handbook/engineering/distribution/onboarding.md:3]: Is there an engineering onboarding guide that we can say they should have completed first?

https://about.sourcegraph.com/handbook/engineering/onboarding

And there's per-role ones...
[Software engineer onboarding](https://about.sourcegraph.com/handbook/engineering/onboarding/software-engineer-onboarding)
[Engineering manager onboarding](https://about.sourcegraph.com/handbook/engineering/onboarding/engineering-manager-onboarding)

File: handbook/engineering/distribution/onboarding.md (hunk 1 above)
Comment (dan-mckean) [reply handbook/engineering/distribution/onboarding.md:21]: Add links too :)

File: handbook/engineering/distribution/onboarding.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+### Week 2
+
+#### Goals:
+- Develop high-level understanding of product.
+- You are participating in our team rituals.
+- Get to know your team.
+- Close at least one issue.
+#### Tasks:
+- Attend product tour.
```
Comment (dan-mckean) [inline handbook/engineering/distribution/onboarding.md:41]: How/when/where do those happen

**Review by dan-mckean — COMMENTED** [review]
Looks good, a few suggestions I'd add in before you merge!

File: handbook/engineering/distribution/onboarding.md (hunk 5 above)
Comment (davejrt) [reply handbook/engineering/distribution/onboarding.md]: Turns out these were stopped in March. I've added links to demos in lieu these sessions

**Pull request merged**